### ISSUE TITLE
ill help

### ISSUE THREAD

**Issue by x8g** [YABOIpy/kahoot-raid#1]
ill continue to work on this, and if I get it working ill make a pull request and release it. Unless you finish it first so yeaaa.

**Comment by YABOIpy** [comment]
Yeah sure go ahead this was just a project from me n my friends I never finished it have fun 😊

**Comment by x8g** [comment]
> Yeah sure go ahead this was just a project from me n my friends I never finished it have fun 😊

Yeah, I was planning on making a working kahoot raider in python a couple weeks ago but I never had the motivation to actually start it.

**Comment by YABOIpy** [comment]
I jus be lazy n got 0 ideas on what to make 😭🤣
